conf/messages

```diff
@@ -720,3 +720,11 @@ prevalidateTrader.consigneeTraderIdentification.hint = The ID contains 13 charac
 prevalidateTrader.consigneeTraderIdentification.error.required = Enter the consignee trader ID
 prevalidateTrader.consigneeTraderIdentification.error.invalidCharacters = Consignee trader ID must only contain letters and numbers
 prevalidateTrader.consigneeTraderIdentification.error.invalidRegex = Consignee trader ID must start with 2 capital letters followed by 11 numbers or mixed letters and numbers
+
+# Prevalidate trader - EPC page
+prevalidateTrader.exciseProductCode.select.defaultValue = Choose excise product code
+prevalidateTrader.exciseProductCode.title = Add excise product codes (EPC)
+prevalidateTrader.exciseProductCode.heading = Add excise product codes (EPC)
```

Comment:
Prototype has "Add an excise product code (EPC)" (singular) - could you double check which is correct? Since we're only adding one at a time I think singular is probably right but it might be a question for UCD just in case.

Reply:
Followed copy deck, but this has now been udpated. So I've changed.